src/pages/Suitcase/Suitcase.tsx

```diff
@@ -1,16 +1,69 @@
-import React, { Component } from 'react';
-// import TopicCard from '../../components/TopicCard';
-import { useSelector } from 'react-redux';
-import { RootState } from '../../store/reducers';
-import { Link } from 'react-router-dom';
+import React from 'react';
 import SuitcaseAnimation from '../../components/Suitcase/SuitcaseAnimation';
+import SuitcaseLabeled from '../../components/Images/labeledSuitcase.png'; 
+import { styles } from './SuitcaseStyles'; 
+import Switch from '@material-ui/core/Switch';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { withStyles } from '@material-ui/core/styles';
+import { red, green } from '@material-ui/core/colors';
+import { Typography } from '@material-ui/core';
+
+const CustomSwitch = withStyles({
+  switchBase: {
+    color: red[700], 
+    '&$checked': {
+      color: green[600],
+    },
+    '&$checked + $track': {
+      backgroundColor: green[500],
+    },
+  },
+  checked: {},
+  track: {
+    backgroundColor: red[800]
+  },
+})(Switch);
+
 
 function Suitcase({ match }) {
+  const title = { //where can I style this nicer? SuitcaseStyles and App.tsx don't work 
```

Comment:
If you want to use SuitcaseStyles you'll want to `export default withstyles(styles)(Suitcase)` !!

Reply:
thank you! this worked